<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefsLogging</class>
 <widget class="QWidget" name="PrefsLogging">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>466</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Logging</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="_log">
     <item>
      <widget class="QLabel" name="AutoLogFileName_label">
       <property name="text">
        <string>Log to:</string>
       </property>
       <property name="buddy">
        <cstring>AutoLogFileName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="AutoLogFileName">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="AutoLogFileName_browse">
       <property name="text">
        <string>&amp;Browse...</string>
       </property>
       <property name="icon">
        <iconset theme="document-open"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_options">
     <item>
      <widget class="QGroupBox" name="_log_format">
       <property name="title">
        <string>Format of log</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="LogFormat_Text">
          <property name="text">
           <string>Text</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="LogFormat_Html">
          <property name="text">
           <string>&amp;HTML</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="LogFormat_Raw">
          <property name="text">
           <string>Raw</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="_what_to_log">
       <property name="title">
        <string>What to log</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="LogOutput">
          <property name="text">
           <string>Log &amp;Output</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="LogInput">
          <property name="text">
           <string>Log &amp;Commands</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="LogNotes">
          <property name="text">
           <string>Log &amp;Notes</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="_log_mode">
       <property name="title">
        <string>Log mode</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="LogMode_Append">
          <property name="text">
           <string>Append</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="LogMode_Overwrite">
          <property name="text">
           <string>Overwrite</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="_session">
     <item row="0" column="0">
      <widget class="QLabel" name="LogFilePreamble_label">
       <property name="text">
        <string>Session &amp;Header:</string>
       </property>
       <property name="buddy">
        <cstring>LogFilePreamble</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPlainTextEdit" name="LogFilePreamble"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="LogFilePostamble_label">
       <property name="text">
        <string>Session &amp;Footer:</string>
       </property>
       <property name="buddy">
        <cstring>LogFilePostamble</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="LogFilePostamble"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="_affixes">
     <item row="0" column="1">
      <widget class="QLabel" name="label_preamble">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Prefix</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2" alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLabel" name="label_postamble">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Suffix</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_output">
       <property name="text">
        <string>Output:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="LogPreambleOutput"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="LogPostambleOutput"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_input">
       <property name="text">
        <string>Commands:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="LogPreambleInput"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="LogPostambleInput"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="Notes_label">
       <property name="text">
        <string>Script:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="LogPreambleNotes"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="LogPostambleNotes"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>LogFormat_Html</sender>
   <signal>toggled(bool)</signal>
   <receiver>_what_to_log</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
